src/components/IKImage/index.tsx

```diff
@@ -138,25 +138,27 @@ const IKImage = (props: Omit<ImageProps, "src" | "loading" | "loader"> & IKImage
   useEffect(() => {
     // if height and width are there in transformation skip props height and width and add fill =true
     const updatedRestProps = restProps;
-    if (
-      transformation?.length &&
-      transformation.some((obj) => obj.hasOwnProperty('height') || obj.hasOwnProperty('width'))
-    ) {
+    if (transformation?.length && transformation.some((obj) => obj.hasOwnProperty("height") || obj.hasOwnProperty("width"))) {
       if (updatedRestProps.height) delete updatedRestProps["height"];
       if (updatedRestProps.width) delete updatedRestProps["width"];
       updatedRestProps["fill"] = true;
     }
     setImageProps(updatedRestProps);
   }, []);
 
+  useEffect(()=>{
+    if(lqip?.active)
+      console.warn("Eager is applied")
```

Comment:
Make the error properly descriptive and add a `[imagekitio-react]` at the start

Something like `[imagekitio-react] paramters lqip and loading cannot be used together`

and show the warning only when loading is passed as a prop

Reply:
done